heartbeat/VirtualDomain

```diff
@@ -518,6 +518,14 @@ VirtualDomain_start() {
 		return $OCF_SUCCESS
 	fi
 
+	# systemd drop-in to stop domain before libvirtd terminates services
+	# during shutdown/reboot
+	if ps -p 1 | grep -q systemd ; then
```

Comment:
I think a better way to detect systemd is to use

`[ "$(cat /proc/1/comm 2>/dev/null)" = "systemd" ]`

Reply:
I think that @beekhof once pointed it should be extracted to a function. I'll refactor the branch accordingly, then the actual detection can be easily modified.